translations.php

```diff
@@ -20,3 +21,6 @@
 // Changed in version X.Y
 // new TranslatableMarkup('Example');
 // new PluralTranslatableMarkup($count, '1 example', '@count examples');.
+// Changed in version 7.2.
```

Comment:
Lets preserve the empty line here for readability. Coding standards tell us not to, but this file is there for documentation so we don't need coding standards. :)
```suggestion

// Changed in version 7.2.
```